packages/stream-caip10-link/src/caip10-link.ts

```diff
@@ -146,9 +145,9 @@ export class Caip10Link extends Stream {
   static makeGenesis(accountId: AccountID): GenesisCommit {
     // Ethereum addresses specifically are sometimes encoded with mixed case and
     // sometimes all lower case. In order to deal with this and not have different
-    // links for different addresses we encode the address using eip55.
+    // links for different addresses we using the address using lowercase.
```

Comment:
```suggestion
    // links for different addresses we convert the address to lowercase.
```

Reply:
"we using the address using lowercase" <- broken grammar

Reply:
lol, I'm tired